.github/files/setup-wordpress-env.sh

```diff
@@ -133,6 +133,43 @@ for PLUGIN in projects/plugins/*/composer.json; do
 	echo "::endgroup::"
 done
 
+# Install WooCommerce plugin used for some Jetpack integration tests.
+# Todo: check the require paths being used by the Woo tests, they may need updating since the Woo repo structure changed.
+# Todo: what to check so this only runs when Jetpack will be tested?
+# Todo: what PHP_VERSION variable is considered "default"?
```

Comment:
Currently the best way to determine that is probably something like
```sh
DEFAULT_PHP_VERSION=$(bash -c 'source .github/versions.sh; echo $PHP_VERSION')
```

Reply:
Thanks, wasn't sure if we were doing something similar elsewhere already 👍 a5075c9